src/protocol/boolean/bitwise_equal.rs

```diff
@@ -46,6 +46,26 @@ where
     no_ones(ctx, record_id, &xored_bits).await
 }
 
+pub async fn bitwise_equal_gf2<C, S>(
+    ctx: C,
+    record_id: RecordId,
+    a: &[S],
+    b: &[S],
+) -> Result<S, Error>
+where
+    C: Context,
+    S: LinearSecretSharing<Gf2> + BasicProtocols<C, Gf2>,
+{
+    debug_assert!(a.len() == b.len());
+    let c = a
+        .iter()
+        .zip(b.iter())
+        .map(|(a_bit, b_bit)| a_bit.clone() - b_bit)
+        .collect::<Vec<_>>();
+
+    no_ones(ctx, record_id, &c).await
```

Comment:
No one's which exactly?

These functions should probably be named `all_` or `any_`.  `no_` was confusing.

Reply:
"no ones" makes sure a list of shares is "all_zeroes". I could call it that instead.